packages/flutter/test/rendering/editable_gesture_test.dart

```diff
@@ -8,9 +8,10 @@ import 'package:flutter/rendering.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
-  setUp(() => _GestureBindingSpy());
+  final TestWidgetsFlutterBinding binding = _GestureBindingSpy();
 
-  test('attach and detach correctly handle gesture', () {
+  testWidgets('attach and detach correctly handle gesture', (_) async {
+    assert(WidgetsBinding.instance == binding);
```

Comment:
nit: in tests it is more idiomatic to use `expect` instead of `assert`, but either will do the job.